cern.root.ROOT.appdata.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+    <id>cern.root.ROOT</id>
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>LGPL-2.1-or-later</project_license>
+    <name>ROOT</name>
+    <summary>Data Analysis Framework</summary>
+    <description>
+	<p>
+	    ROOT is a framework for data processing, developed at CERN,
+            the epicenter of high-energy physics research. It can handle 
+            extensive data using the mathematical and statistical methods 
+            it provides, along with user-developed routines. The results 
+            can be visually presented according to specific needs and 
+            exported to various formats such as PDF, PS, PNG, etc.
+	</p>
+    </description>
+    <screenshots>
+        <screenshot type="default">
+          <image>https://root.cern.ch/root/htmldoc/guides/users-guide/pictures/rootlogo.png</image>
+        </screenshot>
+        <screenshot>
+          <image>https://root.cern/doc/master/eve_aliceproj.png</image>
+        </screenshot>
+        <screenshot>
+          <image>https://root.cern/primer/examplefit.png</image>
+        </screenshot>
+        <screenshot>
+          <image>https://root.cern/doc/v612/pict1_histpalettecolor.C.png</image>
+        </screenshot>
+    </screenshots>
+    <categories>
+        <category>Science</category>
+        <category>Education</category>
+    </categories>
+    <updatecontact>sixpalax_at_gmail.com</updatecontact>
+    <url type="bugtracker">https://github.com/root-project/root/issues</url>
+    <url type="homepage">https://root.cern/</url>
+    <url type="help">https://root-forum.cern.ch/</url>
+    <developer_name>ROOT</developer_name>
+    <kudos>
+        <kudo>HiDpiIcon</kudo>
+        <kudo>HighContrast</kudo>
+        <kudo>UserDocs</kudo>
+    </kudos>
+    <launchable type="desktop-id">root.desktop</launchable>
```

Comment:
Doesn't match desktop id in flatpak

Reply:
I put rename-desktop-file: root.desktop

Reply:
The compilation is successful but it fails to rename the .desktop, I don't know why
Build [92651 failed](https://buildbot.flathub.org/#/builders/6/builds/92651)

Reply:
Ah hmm